config/vufind/facets.ini

```diff
@@ -208,6 +210,17 @@ translated_facets[] = callnumber-first:CallNumberFirst
 ;delimited_facets[] = author_id_str
 ;delimited_facets[] = "author_id_str|:::"
 
+; Sort options for hierarchical facets:
```

Comment:
We have three comments in the file that all say "Sort options for hierarchical facets." It might be helpful to make them more specific to clarify the scope in which each block applies. (I realize that the config section should implicitly explain this, but I don't think it hurts to be a little more explicit).

Reply:
Edited the comments to be more specific.